Documentation/configuration.md

```diff
@@ -270,3 +270,45 @@ and password `gle`.
 Note that _within_ a particular configuration directory (either system or
 local), it is a syntax error for the same Docker registry to be defined in
 multiple files.
+
+### rktKind: `directories`
+
+This kind of configuration is used to customize the various directories that
+rkt uses. The configuration files should be placed inside a `dirs.d` subdirectory
+(e.g. in `/usr/lib/rkt/dirs.d` or `/etc/rkt/dirs.d`).
+
+#### rktVersion: `v1`
+
+##### Description and examples
+
+This version of `directories` configuration specifies one additional field:
+`data`.
```

Comment:
At the moment, we have 3 configurable directories:
```
	--dir=/var/lib/rkt		rkt data directory
	--local-config=/etc/rkt		local configuration directory
	--system-config=/usr/lib/rkt	system configuration directory
```

And this `data` field is to specify only the first one (`--dir`). I wonder if it is obvious enough when reading the documentation that it is only about `--dir`. Is the name of the field `data` clear enough? But I don't have better suggestion.